libs/vm-modules/tests/unit/ml/ml_model_tests.cpp

```diff
@@ -1137,6 +1165,254 @@ TEST_F(VMModelTests, model_sequential_flatten_2d_in_2d_out)
   EXPECT_TRUE(constructed_shape == expected.shape());
 }
 
+TEST_F(VMModelTests, model_sequential_reshape_2d_in_2d_out)
+{
+  static char const *SRC_METRIC = R"(
+          function main() : Tensor
+                          var shape = Array<UInt64>(1);
+                          shape[0] = 1u64;
+                          var x = Tensor(shape);
```

Comment:
this part is almost entirely irrelevant; Perhaps we should modify the empty constructor to instantiate an empty tensor. Then you can just do:
var x = Tensor();
x.fromString(string);

Reply:
Or even better: map tensor fromString constructor to create tensors in etch like so:
var x = Tensor(string);
As far as I remember only constructor from shape is mapped to etch so I need to write this ugly stuff. I think ml tests could be fixed after one of these bindings is implemented

Reply:
that's fine feel free to do it in another PR